sdk/spring/azure-spring-cloud-autoconfigure/CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # Release History
 
+## 2.2.0-beta.1 (Unreleased)
+
+
 ## 2.0.0 (2020-12-30)
+### Breaking Changes
+- Deprecated the `spring.cloud.azure.managed-identity.client-id` property,
+  use `spring.cloud.azure.client-id` to set the managed identity id when using Managed Identity.
```

Comment:
Have you confirmed with @saragluna about this? I remember there are many modules like this.